R/tbl_baseline_chg.R

```diff
@@ -72,26 +72,25 @@ tbl_baseline_chg <- function(data,
   # ---- Type and content checks ----
   check_data_frame(data)
   check_data_frame(denominator)
-  check_string(analysis_variable)
-  check_string(change_variable)
-  check_string(id)
-  check_string(visit)
-  check_string(visit_number)
+  check_not_missing(id)
+  check_not_missing(visit)
+  check_not_missing(visit_number)
+  check_not_missing(analysis_variable)
+  check_not_missing(change_variable)
+  cards::process_selectors(
+    data,
+    by = {{ by }}, id = {{ id }}, visit = {{ visit }}, visit_number = {{ visit_number }},
+    analysis_variable = {{ analysis_variable }}, change_variable = {{ change_variable }}
+  )
```

Comment:
Awesome, thank you so much for the improvements!

Do we already have checks that the id, visit, visit_number, etc are scalar?

Reply:
Those checks were not previously present but I've added them in now